### PR TITLE
Redesign Development Error Pages

### PR DESCRIPTION
**Pull request by cleptric**
I added a bit more space here and there and changed the colours. ~Also, both columns scroll now independently, to provider better context.~

**Before**
Screenshot 2019-05-21 at 18 49 58

**Now**
Screenshot 2019-05-21 at 18 50 04

### CODE REVIEW

**Comment by ADmad** [comment]
I prefer the horizontal separators in left panel (they could be made thinner) and the colors of the error message bar which are present currently :slightly_smiling_face:.

**Comment by ADmad** [comment]
I am fine not having the horizontal separator in left panel but would prefer if the error message and suggestion bar use different primary/base colors and both not be shades of blue.

**Comment by josegonzalez** [comment]
How does this compare to Rails error messages or Whoops? What can we steal from them?

**Comment by cleptric** [comment]
@ADmad I'll provide some different colours to pick from :)
@josegonzalez Whoops looks like this: https://camo.githubusercontent.com/31a4e1410e740fd0ccda128cbcab8723f45e7e73/687474703a2f2f692e696d6775722e636f6d2f305651706539362e706e67

What would be neat, is to have a bit of JS to copy the suggested code to the clip board.
And maybe a refresh of the passed arguments.

**Comment by cleptric** [comment]
Screenshot 2019-05-21 at 21 24 01
@ADmad

**Comment by markstory** [comment]
> I prefer the horizontal separators in left panel (they could be made lighter) and the colors of the error message, suggestion bars which are present currently

Whitespace/margins are a separator :smile:

**Comment by inoas** [comment]
> Screenshot 2019-05-21 at 21 24 01

Very nice aside one thing, the vertical separation between line references in the stack trace is too big...it takes a lot of screen real estate and you can have dozens of lines there.

**Comment by inoas** [comment]
I'd also prefer if it said # LINE / FILE or have a tabular layout aka

ISSUE (colspan 2)
FILE | LINE

**Comment by inoas** [comment]
For the stack I'd also rather see very slight and thin lines than none to be able to remove most of the vertical spacing without adding any kind of ambiguity.

**Comment by markstory** [comment]
@inoas Those are all great suggestions, perhaps we can get them in as a follow up change?

File: templates/layout/dev_error.php [hunk 1]
```diff
@@ -202,96 +224,92 @@
...
+            <div class="error-suggestion">
+                <?= $this->fetch('file') ?>
+            </div>
 
-        <div class="error-suggestion">
-            <?= $this->fetch('file') ?>
+            <?php if ($this->fetch('templateName')): ?>
+            <p class="customize">
+                If you want to customize this error message, create
+                <em><?= APP_DIR . DIRECTORY_SEPARATOR . 'Template' . DIRECTORY_SEPARATOR . 'Error' . DIRECTORY_SEPARATOR . $this->fetch('templateName') ?></em>
```
Comment (markstory) [inline templates/layout/dev_error.php]: This path looks like the 3.x path and not the 4.x one which is

```php
<em><?= 'templates' . DIRECTORY_SEPARATOR . 'Error' . DIRECTORY_SEPARATOR . $this->fetch('templateName') ?></em>
```

**Pull request merged**